app/components/widget-adapter/component.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 import ENV from 'analytics-dashboard/config/environment';
 import stringify from 'npm:json-stable-stringify';
+import tsInterval from '../../utils/date-interval';
```

Comment:
Not really, the consuming page can name it anything they want. I guess there can be a best practice reason to name the utility what it is in import